src/components/CoursesPage/CourseCard.js

```diff
@@ -71,8 +73,10 @@ type Props = {
 };
 
 class CourseCard extends React.Component<Props, State> {
+
   state = {
     palette: {},
+    unenrolling: Boolean
```

Comment:
Me sorprende que esto no rompa todo jaja. Acá se definen los valores default del state, no el type.
```suggestion
    isUnenrolling: false
```

Parece que falta definir el State...

Podés hacerlo abajo de Props antes de definir la clase

```
type State  = {
    palette: any,
    isUnenrolling: Boolean,
}